srcpkgs/splatmoji/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'splatmoji'
+pkgname=splatmoji
+version=1.2.0
+revision=1
+conf_files="/etc/xdg/${pkgname}/${pkgname}.config"
+hostmakedepends="pandoc"
+depends="bash>4.4 rofi xdotool xsel jq"
+checkdepends="jq shunit2"
+short_desc="Emoji and kaomoji selector"
+maintainer="Allen Sobot <chilledfrogs@disroot.org>"
+license="MIT"
+homepage="https://github.com/cspeterson/splatmoji"
+distfiles="https://github.com/cspeterson/${pkgname}/archive/v${version}.tar.gz"
+checksum=cfcc9949ae053be2106fb64cbd75adfff5a772a3b4bc2ae15ad44e2997a0e6ee
+
+do_check() {
+	./test/*
```

Comment:
Just use the full path `./test/unit_tests`

Reply:
Took it from the build scripts, but might be better to be more explicit I guess, yes